### ISSUE TITLE
spliceFunction is used from the builtin environment but is not defined in the Modelica Specification!

### ISSUE THREAD

**Issue by modelica-trac-importer** [modelica/Modelica#488]
**Reported by adrpo on 25 Jan 2011 16:13 UTC**
Model: Modelica.Fluid.Pipes.BaseClasses.HeatTransfer.LocalPipeFlowHeatTransfer
uses function spliceFunction via this call:
```
Nus[i]=spliceFunction(Nus_turb[i], Nus_lam[i], Res[i]-6150, 3850);
```
but this function is nowhere to be found in the Modelica Specification!

Is this a bug and:
 Modelica.Media.Air.MoistAir.Utilities.spliceFunction
should be used instead? Looks like it because all other
uses of spliceFunction is done via Utilities.spliceFunction(...).
 
If this is not a bug then all Modelica tools should support this function?
If so, what's the definition of this function?

Cheers,
Adrian Pop and Per Östlund/

----

Migrated-From: https://trac.modelica.org/Modelica/ticket/488

**Comment by modelica-trac-importer** [comment]
**Comment by hubertus on 25 Jan 2011 16:37 UTC**
Dymola has a built-in spliceFunction, which is why this happened, but it is not standardized and should not be used there. The location in Modelica.Media is not optimal and should be changed. Uses in MSL should use spliceFunction() from inside MSL.

**Comment by modelica-trac-importer** [comment]
**Comment by kurzbach on 2 May 2011 14:06 UTC**
Maybe this should be changed in the MSL.

**Comment by modelica-trac-importer** [comment]
**Comment by hubertus on 2 May 2011 18:18 UTC**
Certainly this should be changed. There is a corresponding Modelica-function. I'll do it as soon as I find time.

**Comment by modelica-trac-importer** [comment]
**Comment by hubertus on 10 May 2011 20:21 UTC**
Fixed in 82f8c794f2f0c93bc91652f98dc6a5b3d5887d6f, 34a7216f76c6cdbda9cc1d16e7d47a778f91ead9 for trunk and 3.1 maintenance. These are the branches which are acutally used at the moment AFAIK, closing ticket therefore.
